plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergUtil.java

```diff
@@ -712,6 +713,14 @@ public static int parseVersion(String metadataFileName)
         if (matcher.matches()) {
             return parseInt(matcher.group("version"));
         }
+        else {
+            // Let's try to parse the version from metadata file name (eg: v1.metadata.json) which
+            // might be generated while creating Hadoop path-based iceberg tables.
```

Comment:
i think this comment is redundant. 
HADOOP_GENERATED_METADATA_FILE_NAME_PATTERN has a good descriptive name